apps/api/src/domain/models/term/tokenization/tokenizer.interface.ts

```diff
@@ -0,0 +1,40 @@
+import { LanguageCode } from '@coscrad/api-interfaces';
+
+/**
+ * TODO[naming] This is tough to name. It's meant to be
+ * either a (possibly length > 1) string of roman characters
+ * that represents either one atomic letter in the target language
+ * or punctuation or white space, or else an out-of-alphabet symbol (e.g. in a loan-word)
+ */
+export type AlphabetCharacters = {
```

Comment:
Let's put some more thought to this in the near future. `Character` would be find if it didn't overlap with a basic data type. `Letter` is not sufficient because we want to include 
- atomic letters in the indigenous language
- out of sequence single-uniqode chars
- punctuation

Reply:
I've updated the comment in the code.